hist/hist/v7/inc/ROOT/TAxis.hxx

```diff
@@ -247,7 +247,7 @@ public:
 private:
    unsigned int fNBins;         ///< Number of bins including under- and overflow.
    std::string fTitle;          ///< Title of this axis, used for graphics / text.
-   const bool fCanGrow = false; ///< Whether this axis can grow (and thus has no overflow bins).
+   bool fCanGrow = false;       ///< Whether this axis can grow (and thus has no overflow bins).
```

Comment:
@axel  Was the 'const' intentional to mark that the state can not change?  Should the operator= be deleted instead?  And/or should a comment be added to replace the 'const'?

Reply:
That should be a `static constexpr const bool fCanGrow`, please.

Reply:
@Axel-Naumann That sounds odd.  Such a change need to be followed by the removal/tweak of
```
   TAxisBase(std::string_view title, int nbinsNoOver, bool canGrow) noexcept
      : fNBins(nbinsNoOver + (canGrow ? 0 : 2)), fTitle(title), fCanGrow(canGrow)
{}
```
or am I looking in the wrong place?